pilot/model/config.go

```diff
@@ -471,6 +501,12 @@ func SortRouteRules(rules []Config) {
 }
 
 func (store *istioConfigStore) RouteRules(instances []*ServiceInstance, destination string) []Config {
+	configs := store.routeRules(instances, destination)
+	configs = append(configs, store.routeRulesV2(instances, destination)...)
+	return configs
+}
+
+func (store *istioConfigStore) routeRules(instances []*ServiceInstance, destination string) []Config {
```

Comment:
Add a TODO here ..
//TODO: remove once transition is complete.

We are not going to support both v1alpha1 and v1alpha2 simultaneously.

Reply:
I opted not to put any of the TODOs for the transition into this PR because I thought it would be too verbose. I can add them, but I think almost every function I added will need a TODO.